.github/workflows/linux_test.yml

```diff
@@ -0,0 +1,48 @@
+name: "Linux test"
+on: [push]
+
+jobs:
+  documentation_test:
+    runs-on: ubuntu-latest
+    steps:
+    # Setup
+    - uses: actions/checkout@v1
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
+
+    # before_install
+    - run: npm install -g yarn@latest
+
+    # install
+    - run: yarn install --frozen-lockfile
+    - run: yarn run bootstrap
+
+    # script
+    - run: yarn run test:docs
+    - run: yarn run website
+
+  test:
+    strategy:
+      matrix:
+        node: [8.x, 10.x, 12.x]
```

Comment:
Can you add `os: [macOS-10.14, windows-2016, ubuntu-18.04]` to `matrix:` like this?

https://github.com/textlint/textlint/issues/632#issuecomment-544221780